docs/use/myst.md

```diff
@@ -227,6 +227,71 @@ which automatically numbers the labelled objects.
 For example, `` {numref}`my_figure` `` will produce a reference like:
 {numref}`my_figure`.
 
+### Figure scaling and alignment
+
+Figures can also be aligned by using the option `:align: right` or `:align: left`. By default, figures are aligned to the center (see {numref}`my_figure`).
+
+````
+```{figure} ../images/C-3PO_droid.png
+---
+scale: 25%
+align: left
+---
+Here is my figure caption!
+```
+````
+
+```{figure} ../images/C-3PO_droid.png
+---
+scale: 25%
+align: left
+---
+Here is my figure caption!
+```
+
+````
+```{figure} ../images/C-3PO_droid.png
+---
+scale: 25
+align: right
+---
+Here is my figure caption!
+```
+````
+
+```{figure} ../images/C-3PO_droid.png
+---
+scale: 25
+align: right
+
+---
+Here is my figure caption!
+```
+
+### Figure Parameters
+
+The following options are supported:
+
+* `scale` : _integer percentage_
+
+  The scale paramater helps to uniformly scale the figure. The default is "100" which indicates no scaling. The symbol "%" is optional
+
+* `width` : _length or percentage_
+  You can set the figure width in the following units: "em","ex","px","in","cm","mm","pt","pc","%"
+
+* `height` : _length_
+  You can set the figure height in the following units: "em","ex","px","in","cm","mm","pt","pc",""
+
+* `alt` : _text_
+  This paramater is useful to users who might not be able to see the figure and it generally entails a short description of the figure
+
+* `align` : _"left", "center", or "right"_
+  Default alignment is center
+
+* `name` : _text_
+
+* `class` : _text_
```

Comment:
should this be `figclass` instead of `class`? I'm not sure I know the difference between them but `figclass` is what I've seen elsewhere.

Reply:
I think you're right but I have to double check. `figclass` is used for figure directives while `class` is used for image directives. I've seen `figclass` in the context of aligning figures, e.g.: `:figclass: align-center`. I'll check it out and report tomorrow.